scrapers/GGW.yml

```diff
@@ -0,0 +1,26 @@
+name: "Girls Gone Wild"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - girlsgonewild.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[contains(@class,"watch__title")]/text()
+      Details: //div[contains(@class,"text-mob-more")]/text()
+      Performers:
+        Name: //span[contains(@class,"watch__featuring_models")]/a
+      Studio:
+        Name: "Girls Gone Wild"
+        URL: "https://girlsgonewild.com"
```

Comment:
Change to
```yaml
      Studio:
        Name: //div[@class='p-md text-light mb-5 genres-list']/a
        URL: //div[@class='p-md text-light mb-5 genres-list']/a/@href
```

They have multiple studios listed. https://girlsgonewild.com/studios